client/src/components/FilesManager/AddFolder.jsx

```diff
@@ -72,7 +72,7 @@ const AddFolder = ({ show, onClose, projectID, parentDirectory, onFinishedAdd })
     if (validateForm()) {
       setLoading(true);
       try {
-        const createRes = await axios.post(`/project/${projectID}/book/materials/`, {
+        const createRes = await axios.post(`/project/${projectID}/files/addFolder`, {
```

Comment:
am i missing the backend implementation for this? Is there an explicit need for a separate endpoint for folder creation?